src/Symfony/Component/DependencyInjection/Argument/TaggedIteratorArgument.php

```diff
@@ -41,6 +42,7 @@ public function __construct(string $tag, string $indexAttribute = null, string $
         $this->indexAttribute = $indexAttribute;
         $this->defaultIndexMethod = $defaultIndexMethod ?: ('getDefault'.str_replace(' ', '', ucwords(preg_replace('/[^a-zA-Z0-9\x7f-\xff]++/', ' ', $indexAttribute ?? ''))).'Name');
         $this->needsIndexes = $needsIndexes;
+        $this->defaultPriorityMethod = $defaultPriorityMethod;
```

Comment:
`$this->defaultPriorityMethod = $defaultPriorityMethod ?: ('getDefault'.str_replace(' ', '', ucwords(preg_replace('/[^a-zA-Z0-9\x7f-\xff]++/', ' ', $indexAttribute ?? ''))).'Priority');`

Reply:
For what I understand, it's used only to extrapolate the method to get the key (for `defaultIndexMethod`). here it's absolutely not related isn't ?

More over, this will not work since it will generate something like `getDefaultPriority`. But the default priority is `0` and we don't want to force user to implement a `getDefaultPriority` method.

Finally, there is not relation between a key and a priority (IMHO).

Or maybe I missed something

Reply:
This would allow defining the priority using a method that matches the convention.
- `!tagged_iterator my.foo` => defining `static function getDefaultPriority()` and the code can decide for it
- `!tagged_iterator { tag: my.foo index_by: foo }` => defining `static function getDefaultFooPriority()` does the same
I think it would be nice.

Reply:
I discussed with @nicolas-grekas and he win even if I still disagree.
The main argument is: we should ne consistant with how `default_index_method` works.